lib/messageReceiver.ts

```diff
@@ -528,12 +532,31 @@ export class MessageReceiver extends LinkEntity {
     }
   }
 
+  /**
+   * Disposes the message with the specified disposition.
+   * @param delivery Delivery associated with the message.
+   * @param operation The disposition type.
+   * @param [options] optional parameters that can be provided while disposing the message.
+   */
   async disposeMessage(delivery: Delivery, operation: DispositionType, options?: DispositionOptions): Promise<any> {
```

Comment:
Didn't notice this before, but `disposeMessage` is a bit of a weird name. "dispose" and "disposition" don't really mean the same thing. I'd probably call this `settleMessage` or something like that